src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from 'react-i18next';
+import { useFetch } from '../../../../utils/hooks/useFetch';
+import { Registration } from '../../../../types/registration.types';
+import { SearchApiPath } from '../../../../api/apiPaths';
+import { SearchResponse } from '../../../../types/common.types';
+import { ListSkeleton } from '../../../../components/ListSkeleton';
+import { Divider, List, Typography } from '@mui/material';
+import { CentralImportResultItem } from './CentralImportResultItem';
+import { ResourceFieldNames } from '../../../../types/publicationFieldNames';
+import { createSearchQuery, ExpressionStatement, SearchConfig } from '../../../../utils/searchHelpers';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../../utils/constants';
+
+interface CentralImportDuplicateSearchProps {
+  publication: Registration;
+}
+export const CentralImportDuplicateSearch = ({ publication }: CentralImportDuplicateSearchProps) => {
```

Comment:
hele denne er egentlig bare en placeholder for neste task som skal legge inn et skjema med søkemuligheter. Det samme gjelder presentasjonen av søkeresultatene. Poenget var bare å vise at man kan initielt vise et søk på en doi